### PR TITLE
Added Instruction for windows Users and updated requirements.txt 

### PR DESCRIPTION
**Pull request by tusharrao198**
- I  updated the method to handle the sensitive data. We can now access the details from a external file which can be added to gitignore to avoid uploading of senstive data.  

- Added Instructions for windows users to setup the project.

- Updated the requirements.txt file which can be useful ahead.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -2,44 +2,86 @@
...
    1. You should have postgresql installed on your system. If not, run
-    
+      **On Linux** -
         `sudo apt update`
 
         `sudo apt install postgresql postgresql-contrib`
-    2. Create a local postgres database. Keep the credentials handy, you'll need them to link the database to django.
+
+      **On Windows** - First Install- 1. PostgreSQL - `https://www.postgresql.org/download/windows/`
+                                 **( NOte :- Remeber Password entered here.)**
```
Comment (LalitNM) [inline Contributing.md:15]: 'O' is in upper case, correct this.

**Review by LalitNM — CHANGES_REQUESTED** [review]
And also resolve conflicts by rebasing your local branch. You have to read about `git rebase upstream` and `git rebase -i`.

File: Contributing.md (hunk 1 above)
Comment (tusharrao198) [reply Contributing.md]: Corrected

**Comment by harnaman-hk** [comment]
@tusharrao198 It seems that some formatting is required. Can you fix this?


Screenshot from 2020-10-05 16-26-52

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
@tusharrao198 you have to resolve conflicts so that I can merge this PR
Steps which may help you

- Reset this branch to a state before creating this PR.

- add remote upstream to your local git repository using `git remote add upstream <link to upstream>`

- rebase this branch to upstream: master using `git rebase upstream/master`

- add your changes in one commit only

- force push this branch here using `git push -f origin: master`

And also read about reset, upstream, rebase, force-push.

**Comment by tusharrao198** [comment]
> @tusharrao198 you have to resolve conflicts so that I can merge this PR
> Steps which may help you
> 
> * Reset this branch to a state before creating this PR.
> * add remote upstream to your local git repository using `git remote add upstream <link to upstream>`
> * rebase this branch to upstream: master using `git rebase upstream/master`
> * add your changes in one commit only
> * force push this branch here using `git push -f origin: master`
> 
> And also read about reset, upstream, rebase, force-push.

OK wiil do!

**Comment by tusharrao198** [comment]
Resloved conflict finally..

**Comment by LalitNM** [comment]
Merged :rocket:.

**Pull request merged**